telemetry/telemetryformat.md

```diff
@@ -42,19 +45,68 @@ _metrics_ is a required field that contains 1 or more entries
 ]
 ```
 
-These are then used to generate functions that take arguments pertaining to metrics. An example of one of these generated functions is:
+These are then used to generate functions that take arguments pertaining to metrics. For a concrete example:
```

Comment:
Is there supposed to be an example in the next line?

Reply:
I shuffled this file around, fixed